packages/adena-extension/.eslintrc.json

```diff
@@ -17,11 +17,12 @@
     "ecmaVersion": "latest",
     "sourceType": "module"
   },
-  "plugins": [
-    "react",
-    "@typescript-eslint"
-  ],
-  "rules": {},
+  "plugins": ["react", "@typescript-eslint"],
+  "rules": {
```

Comment:
You changed almost all the double quotes to single quotes.
I think it would be nice to add a `single` quote option to the eslint rule.

Additionally, it would be nice to have a `no-unused-vars` option.